fleet/sg.tf

```diff
@@ -1,51 +1,61 @@
 resource "aws_security_group" "ae-nodes" {
-  count = "${(var.spot_nodes + var.static_nodes) > 0 ? 1 : 0}"
-  name  = "ae-${var.env}-nodes-terraform"                      #postfix with terraform to avoid issues with current setup
+  name   = "ae-${var.env}-nodes"
+  vpc_id = "${var.vpc_id}"
 
   lifecycle {
     create_before_destroy = true
   }
-
-  vpc_id = "${var.vpc_id}"
-
-  tags = {
-    Name = "ae-${var.env}-nodes-terraform"
-  }
 }
 
-resource "aws_security_group_rule" "allow_all_internal" {
-  count     = "${(var.spot_nodes + var.static_nodes) > 0 ? 1 : 0}"
-  type      = "ingress"
-  from_port = 0
-  to_port   = 65535
-  protocol  = "TCP"
-
+resource "aws_security_group_rule" "local_in" {
+  type                     = "ingress"
+  from_port                = 0
+  to_port                  = 65535
+  protocol                 = "TCP"
   security_group_id        = "${aws_security_group.ae-nodes.id}"
   source_security_group_id = "${aws_security_group.ae-nodes.id}"
 }
 
-resource "aws_security_group_rule" "external_api_port" {
-  count             = "${(var.spot_nodes + var.static_nodes) > 0 ? 1 : 0}"
+resource "aws_security_group_rule" "external_api_in" {
   type              = "ingress"
   from_port         = 3013
-  to_port           = 3015
+  to_port           = 3013
   protocol          = "TCP"
   cidr_blocks       = ["0.0.0.0/0"]
   security_group_id = "${aws_security_group.ae-nodes.id}"
 }
 
-resource "aws_security_group_rule" "external_healthz_port" {
-  count             = "${(var.spot_nodes + var.static_nodes) > 0 ? 1 : 0}"
+resource "aws_security_group_rule" "internal_api_in" {
+  count             = "${var.enable_internal_api ? 1 : 0}"
   type              = "ingress"
-  from_port         = 8080
-  to_port           = 8080
+  from_port         = 3113
+  to_port           = 3113
+  protocol          = "TCP"
+  cidr_blocks       = ["0.0.0.0/0"]
```

Comment:
It seems this 3113 port was not configured before. And 0.0.0.0? Is this intended?

Reply:
> - allow enabling of internal and SC websocket APIs

Seems yes.